src/gobstuf/stuf/brp/response_mapping.py

```diff
@@ -757,7 +757,12 @@ def filter(self, mapped_object: dict, **kwargs):
             # Compares only the same number of characters as the given string, to match precision
             return None
         elif not naam.get('geslachtsnaam') and not naam.get('voornamen') and not mapped_object.get('geboorte'):
```

Comment:
Maybe replace with something like `elif not any([name.get(attr)) for attr in [....])`? Bit easier to read

Reply:
Now that I opened the rest of the code above, there's some more room for improvement here. All the `mapped_object.get()` calls with or without negations with all the checks are a bit hard to read.

I'd replace the (el)if expressions for short function calls, where the function names are descriptive of the check. Also removes the need for the extensive comments, makes the code much easier to read.

Again; no need to change that in this PR, but something to consider in the future.

Reply:
yup, I agree, also trying to omit negations if possible